test/unit/component.test.js

```diff
@@ -111,6 +112,16 @@ test('should do a deep merge of child options', function(){
   Component.prototype.options_ = null;
 });
 
+test('should init child components from component options', function(){
+  var comp = new ControlBar(TestHelpers.makePlayer(), {
```

Comment:
Yeah, I was just being lazy :P
I guess I should make a couple of custom components.